ietf/group/views.py

```diff
@@ -737,26 +737,35 @@ def dependencies(request, acronym, group_type=None):
 
     references = Q(
         Q(source__group=group) | Q(source__in=cl_docs),
-        source__type="draft",
+        source__type="draft",  # does this need to be type__in=["draft", "rfc"]? Check for informative ref rfc->draft
         relationship__slug__startswith="ref",
     )
-
-    both_rfcs = Q(source__type_id="rfc", target__type_id="rfc")
+    rfc_or_subseries = {"rfc", "bcp", "fyi", "std"}
+    both_rfcs = Q(source__type_id="rfc", target__type_id__in=rfc_or_subseries)
+    pre_rfc_draft_to_rfc = Q(source__type_id="draft", source__states__slug="rfc", target__type_id__in=rfc_or_subseries)
+    rfc_to_pre_rfc_draft = Q(source__type_id__in=rfc_or_subseries, target__type_id="draft", target__states__slug="rfc")
+    both_pre_rfcs = Q(
+        source__type_id="draft", source__states__slug="rfc",
+        target__type_id="draft", target__states__slug="rfc",
+    )
```

Comment:
it's probably worth adding that you're looking for `states__type="draft"` to all of the places you are looking for `states__slug=="rfc"` in case some other state machine ever gets an `rfc` node.

Reply:
done